script.js

```diff
@@ -103,3 +103,140 @@ CalculatedDogAge(5);
 CalculatedDogAge(8);
 CalculatedDogAge(4);
 
+// Домашнаяя работа Nr.19
+
+function displayObjectProperties(obj){
+    for (const key in obj){
+      console.log(`${key}: ${obj[key]}`);
+    }
+}
+const myObjekt = {name:'John', age:30, city: 'New-York'};
+displayObjectProperties(myObjekt);
+
+// 2 Задание
+
+function keyMyProject(object, keyNewCheck){
+    for (const key in object){
+        if(key === keyNewCheck){
+          return true;
+        }
+      return false;
+    }
+
+
+    const myNewObject = {name: 'John', age: 30, city: 'New-York'} 
+    const keyToCheck = 'age';
+
+    const result = keyMyProject(myNewObject, keyNewCheck);
+    console.log(result);
+}
+
+
+// 3 Задание
+
+function concatenateObject(objs){
+  let resultone = '';
+
+  const values = Object.values(objs);
+
+  let i=0;
+  do{
+    resultone +=values[i];
+    i++;
+  } while (i < values.length);
+
+  return resultone;
+
+}
+
+  const myObject = { prop1: 'Hello', prop2: 'World', prop3: '!' };
+  const concatenatedValue = concatenateObject(myObject);
+
+  console.log(concatenatedValue);
+
+  // 4 задание
+
+  function countVowels(str) {
+    str = str.toLowerCase();
+  
+    let count = 0; 
+    let i = 0; 
+  
+    while (i < str.length) {
+      const char = str[i];
+  
+ 
+      if (char === 'a' || char === 'e' || char === 'i' || char === 'o' || char === 'u') {
```

Comment:
good solution. It works. But for a limited amount of elements. Ok you have 5 vowels here and you can compare the char with each of them. But what if there are hundreds of elements to compare with? 
So I suggest you to put all the vowels into an array and check if char is in array or not. Same as we did on our practice lesson last week